### DISCUSSION TITLE
来香港办香港银行卡 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#37] [Announcements]
# 来香港办香港银行卡 - forecho's Blog

引言
公司让我们来香港出差，我顺便办了香港银行卡，这里记录一下办卡的过程。
中国银行香港卡
经过
我是从深圳北坐高铁过来的，早上到达香港，出站之后，就百度附近的中国银行，最近的一家在中港城，走十几分钟就到了。
我们大概 10 点半左右到了中国银行香港营业厅，门口取号，给我的是 70 号，目前正在办的是 9 号，一个号大概要办理 45 分钟，工作人员在取号的之前都会告诉你，让你知道大概要等多久。
他们 17 点就下班了，70 号意味着你今天都不一定能办到卡，没想到工作日来香港办卡人都这么多，我看今天大概率没希望了然后我们就走了。
第二天我去酒店附近的分行办卡，人很少，去了就差不多可以办卡了，所以我建议大家不要去中港城的中国银行香港分行办，因为那边人太多了。
我最后是在湾仔地铁口那边的中国银行香港分行办的卡。你也可以去铜锣湾那边的中国银行香港分行办，这两个分行都是在地铁站附近，很方便。一个分行不行的话，你可以去另一个分行办。

https://blog.forecho.com/coming-to-hong-kong-to-get-a-hong-kong-bank-card.html

**Comment by zxlflash** [comment]
"你的通讯地址和现住地址必须一致，如果不一致，你需要提供地址证明"这一条的地址证明指的就是水电费或信用卡账单的证明吧？

**Reply by forecho** [reply]
是的
